pom.xml

```diff
@@ -63,13 +63,28 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
-          <version>3.7.0</version>
+          <version>3.8.0</version>
+          <executions>
+            <execution>
+              <id>default-compile</id>
+              <configuration>
+                <release>9</release>
```

Comment:
Seems like this requires JDK 9 for compilation? 
I can no longer compile with JDK 8. I get this error:
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:compile (default-compile) on project gson: Fatal error compiling: invalid flag: --release -> [Help 1]

How do I fix this?

Reply:
Module info require JDK 9 or newer. While 8 isn't EOL yet, it will be in 5 months. You can grab a JDK 11 build here: https://adoptopenjdk.net/

Reply:
Thanks Jake. I have not upgraded beyond JDK 8 so far because of my focus on Android. 
I guess time to upgrade!

Reply:
Have you used JDK 9 or 10 with Android? And does it work?

Reply:
JDK 11 is my default, yes. It mostly just works fine. And wherever it doesn't it's a bug in the tooling.